### ISSUE TITLE
Add option to use Parchment mappings

### ISSUE THREAD

**Issue by ExpensiveKoala** [architectury/template-generator#12]
Having an option for using parchment mappings would be a great feature for the generator.

**Comment by BrailleBennett** [comment]
Agreed, mojmap layered with parchment is what Neo recommends, so this should be added, plus, it would help raise awareness for new devs that parchment exists.
